requirements-dev.lock

```diff
@@ -6,6 +6,7 @@
 #   features: []
 #   all-features: false
 #   with-sources: false
+#   generate-hashes: false
```

Comment:
Strange. Maybe this was added because the Windows and Python versions will have different hashes now?

Reply:
I think so. we changes the package by platform OS and it would cause the difference of dependencies tree. so I'll ignore it for now.